src/Tests/OneExplorer/ConfigObject.test.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+
+import {ConfigObj} from '../../OneExplorer/ConfigObject';
+
+class TestFileBuilder {
+  testDir: string = '/tmp/one-vscode.test/ConfigObject';
+  fileList: string[] = [];
+
+  constructor(testDir?: string) {
+    if (testDir) {
+      this.testDir = testDir;
+    }
+
+    try {
+      if (!fs.existsSync(this.testDir)) {
+        fs.mkdirSync(this.testDir, {recursive: true});
+      }
+    } catch (e) {
+      console.error('Cannot create temporal directory for the test');
+      throw e;
+    }
+  }
+
+  getPath(fileName: string) {
+    return `${this.testDir}/${fileName}`;
+  }
+
+  writeFileSync(fileName: string, content: string) {
+    const filePath = `${this.testDir}/${fileName}`;
+
+    try {
+      fs.writeFileSync(filePath, content, 'utf-8');
+      console.log('File creation succeeded!');
+    } catch (e) {
+      console.error('Cannot create temporal files for the test');
+      throw e;
+    };
+  }
+
+  destructor() {
+    fs.rmdirSync(this.testDir, {recursive: true});
+  }
+};
+
+suite('OneExplorer', function() {
+  suite('ConfigObject', function() {
+    suite('#createConfigObj()', function() {
+      test('Returns null when file read failed', function() {
```

Comment:
I think this is negative test. It would be better to use different naming to make SE process easier later.

In backend repo, we use `NEG: ` prefix for negative test like the following:

```suggestion
      test('NEG: Returns null when file read failed', function() {
```

Reply:
Did you maybe forgot to change this? 😸

Reply:
Oh sure. I will do this now 😄